submissions/vl-shevchenko/html-css-popup/style.css

```diff
@@ -0,0 +1,268 @@
+html {
+    box-sizing: border-box;
+    font-size: 16px;
+    background-image: url(popup-icons2/Light.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  }
+  
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  body, h1, h2, h3, h4, h5, h6, p, ol, ul {
+    margin: 0;
+    padding: 0;
+    font-weight: normal;
+  }
+  
+  ul, li {
+    list-style: none;
+  }
+
+a:link,
+a:visited  {
+      color: inherit;
+      text-decoration: none;
+  }  
+
+body {
+    height: 100vh;
+    
+}
+
+img {
+  width: 80%;
+}
+
+.conteiner_menu {
+    display: flex;
+    justify-content: flex-end;
+    position: relative;
+    align-items: center;
+    max-width: 1020px;
+    height: 70px;
+    margin: auto;
+    background: rgba(30, 106, 219, 0.616);
+    border: 0.5px solid rgb(245, 239, 235 0.4);
+    border-bottom-right-radius: 28px;
+    border-bottom-left-radius: 28px;
+}
+
+.conteiner__unlist {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-right: 50px;
+    
+}
+
+#checkbox_btn {
+    appearance: none;
+    position: absolute;
+}
+
+.list-link{
+    display: flex;
+    align-items: center;
+    padding: 0px 15px;
+    height: 55px;
+    font-size: 20px;
+    text-align: center;
+    color: #242220;
+    border-radius: 10px;
+    cursor: pointer;
+    justify-content: space-around;
+}
+
+
+
+
+
+.list-link:hover {
+    border-bottom: 3px solid rgb(61, 68, 71);
+    background: rgba(230, 228, 227, 0.199);
+    border-radius: 10px;
+    font-size: 20px;
+    
+}
+
+
+.list-link:focus  {
+    background: rgba(241, 241, 241, 0.74);
+    border-radius: 10px;
+    border-bottom: 3px solid rgb(61, 68, 71);
+    
+}
+
+
+#checkbox_btn:focus ~ label > a {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css). Avoid using too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).